src/modules/site-v2/base/views/tools/pairwise_indel_finder.py

```diff
@@ -249,9 +250,17 @@ def submit():
 
 # TODO: Move internals of this to a service function
 @pairwise_indel_finder_bp.route("/report/<id>")
-@pairwise_indel_finder_bp.route("/report/<id>/tsv/<filename>")
+@pairwise_indel_finder_bp.route("/report/<id>/download/<file_ext>")
```

Comment:
subjective API design -- putting the RESTful hat here, the file_extension might a query param instead of part of the path since it is the same resource, just a different expression. Eg: /download?format=csv